core/tests/puppeteer-acceptance-tests/user-utilities/question-admin-utils.js

```diff
@@ -25,8 +25,8 @@ const contributorDashboardAdminUrl =
 const { showMessage } = require(
   '../puppeteer-testing-utilities/show-message-utils.js');
 
-const reviewQuestionRightValue = 'string:question';
-const submitQuestionRightValue = 'string:submit_question';
+const reviewQuestionRightValue = 'question';
+const submitQuestionRightValue = 'submit_question';
```

Comment:
Why are we making this change? (Asking just so we are aware of the reason!)

Reply:
`'string:'` is a built in in Angular JS for differentiating strings from other data types. But it's not a built in Angular.

When I ran the acceptance test, I found that the select option in Add Contribution Rights wasn't working, thats when I thought that there might be a potential cause in string declaration.

So, I changed it and ran the test. And it Passed!

Reply:
Is it resolved sir?

Reply:
Yes, thanks for the explanation! I have asked @chris7716 to quickly review this PR to avoid further delay!

Reply:
Done.